### PR TITLE
Bookstore: add reducers and actions

### PR DESCRIPTION
**Pull request by Nyame-Wolf**
In this pull request:
- Installed redux
-  Added book's actions and reducer
-  Added category's actions and reducer
- Configured the redux store

### CODE REVIEW

File: src/redux/categories/categories.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// Actions
+const STATUS = '/src/redux/categories/STATUS';
+
```
Comment (codecaiine) [inline src/redux/categories/categories.js]: - [ ] Good job so far Please make sure that your actions follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), according to that you have action types in the form` my-app/reducer/ACTION_TYPE` You have something like `const STATUS = 'Nyame-Bookstore/categories/STATUS'`. Please fix it :+1:

File: src/redux/books/books.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// Actions
+const REMOVE = '/src/redux/books/REMOVE';
+const ADD = '/src/redux/books/ADD';
```
Comment (codecaiine) [inline src/redux/books/books.js]: - [ ] Nice job Please make sure that your actions follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux) according to that you have action types in the form` my-app/reducer/ACTION_TYPE` You have something like `const REMOVE = 'Nyame-Bookstore/books/REMOVE'` and `const ADD = 'Nyame-Bookstore/books/ADD'`. Please fix it :+1: 
- [ ] From the project requirements you should Set an initial state with an empty array then creates and Exports **Action Creators**. (`addBook` and `remove book` functions) , you export well your actions creators but can see you array . Please fix :+1:  find below an example of implementation :arrow_down: 

```
/ ACTIONS
const ADD_BOOK = 'bookstore/books/ADD_BOOK';
const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';

const initialState = [];

// REDUCER
const booksReducer = (state = initialState, action) => {
  switch (action.type) {
    case ADD_BOOK:
      return [...state, action.paylod];
    case REMOVE_BOOK:
      return [...state.filter((book) => book.id !== action.payload.id)];
    default:
      return state;
  }
};

```

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @Nyame-Wolf , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Descriptive PR :heavy_check_mark: 
- Good project implementation :heavy_check_mark: 
- No linters errors :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @Nyame-Wolf 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Your linters are passing all the checks
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**